arviz/stats/stats.py

```diff
@@ -1341,42 +1363,26 @@ def summary(
     # Combine metrics
     metrics: List[xr.Dataset] = []
     metric_names: List[str] = []
```

Comment:
these two lines are resetting the metrics and metric_names lists, so all the stats that were added to them above are lost.